stale_repos.py

```diff
@@ -191,10 +195,12 @@ def write_to_markdown(inactive_repos, inactive_days_threshold, file=None):
             f"The following repos have not had a push event for more than "
             f"{inactive_days_threshold} days:\n\n"
         )
-        file.write("| Repository URL | Days Inactive | Last Push Date |\n")
-        file.write("| --- | --- | ---: |\n")
-        for repo_url, days_inactive, last_push_date in inactive_repos:
-            file.write(f"| {repo_url} | {days_inactive} | {last_push_date} |\n")
+        file.write("| Repository URL | Days Inactive | Last Push Date | Visibility |\n")
+        file.write("| --- | --- | --- | ---: |\n")
+        for repo_url, days_inactive, last_push_date, private in inactive_repos:
+            file.write(
+                f"| {repo_url} | {days_inactive} | {last_push_date} | {private} |\n"
```

Comment:
```suggestion
                f"| {repo_url} | {days_inactive} | {last_push_date} | {visibility} |\n"
```

Reply:
Added fix and set you as co-author.  Thank you for feedback